pilot/pkg/xds/ads.go

```diff
@@ -935,14 +921,12 @@ func (conn *Connection) send(res *discovery.DiscoveryResponse) error {
 				conn.proxy.WatchedResources[res.TypeUrl].LastSize = sz
 			}
 			conn.proxy.Unlock()
+		} else {
```

Comment:
if there is err, it is not always timeout

Reply:
It is like that from the beginning. I corrected it now.